ext/reflection/tests/ReflectionConstant_isDeprecated.phpt

```diff
@@ -0,0 +1,27 @@
+--TEST--
+ReflectionConstant::isDeprecated()
+--EXTENSIONS--
+zend_test
+--FILE--
+<?php
+
+$reflectionConstant = new ReflectionConstant('ZEND_CONSTANT_A');
+var_dump($reflectionConstant->isDeprecated());
+
+$reflectionConstant = new ReflectionConstant('ZEND_TEST_DEPRECATED');
+var_dump($reflectionConstant->isDeprecated());
+
+define('RT_CONST', 42);
+$reflectionConstant = new ReflectionConstant('RT_CONST');
+var_dump($reflectionConstant->isDeprecated());
+
+const CT_CONST = 43;
+$reflectionConstant = new ReflectionConstant('CT_CONST');
+var_dump($reflectionConstant->isDeprecated());
+
+?>
+--EXPECT--
+bool(false)
+bool(true)
```

Comment:
How does this end up being `true`? Where is it defined?

Reply:
It's defined in the test extension: https://github.com/php/php-src/blob/d40726670fd2915dcd8076738c56596e7c877b58/ext/zend_test/test_arginfo.h#L518

Reply:
Ah, it already existed. I should have looked beyond just this PR.